src/insurance/Home.java

```diff
@@ -23,15 +27,19 @@ public int getHomeID() {
         return homeID;
     }
 
+    public double getValue() {
```

Comment:
Value is the value of the home? We should probably store this in the DB too. I don't think we are currently as we didn't know what was needed for calculating risk. Just making a note here for that.